site/_data/extensionSamples.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @fileoverview Extract browser extension samples from sample list data.
+ */
+
+const path = require('path');
+const fs = require('fs');
+
+// The names of the samples matching the list will be hidden.
+const IGNORED_SAMPLES = [
+  'sample.co2meter', // WebHID is not released yet.
+];
```

Comment:
That thought ocurred to me, but I wasn't going to say anything since I'm not an engineer. Also, how else would we get this information?

Reply:
Maybe it's possible to add a metadata in the sample (e.g. a field in the extension-samples.json file) that tells this script whether this sample should be ignored or not. This way you edit "sample is hidden" field at the same time when you edit the sample instead of a separate repo.